SU-SWS/stanford_basic
# D8CORE-3502: Hotfix for active, focus pseudoclasses on dropdown toggle

# READY FOR REVIEW

# Summary
- This is a hotfix to address the immediate priority identified in D8CORE-3502:  'Focus indicator (underline) is missing on down arrow (Chevron) in main menu when tabbing in Chrome. That is, make sure focus, hover, and active states are addressed. '

# Review By (Date)
- ASAP

# Urgency
- blocks releasing dropdowns checkbox for general use.

# Steps to Test

1. Checkout this branch.
2. Set up a drop-down menu structure in your pages
3. Clear your drupal caches
4. Test moving through the nav menu with keyboard. notice the active, focus states for the dropdown toggle correctly get underlined.
